test/react/hooks/use-chat-client.test.tsx

```diff
@@ -50,6 +50,23 @@ describe('useChatClient', () => {
     render(<TestProvider />);
   });
 
+  it('should get the chat client from the context without error and ui kit agent if set', () => {
+    (globalThis as Record<string, unknown>).__ABLY_UI_KIT_VERSION__ = '1.0.0';
+    const chatClient = newChatClient();
+    const TestProvider = () => (
```

Comment:
i think this test isn't extensive enough, is it possible to test that the agent actually gets sent over the wire. it's possible with this kind of set up that the client tries to connect before the window var is populated, in which case this test might pass but in reality the agent still doesn't get sent